app/code/Magento/ConfigurableProduct/Pricing/Price/LowestPriceOptionsProvider.php

```diff
@@ -37,19 +38,37 @@ class LowestPriceOptionsProvider implements LowestPriceOptionsProviderInterface
      */
     private $linkedProductMap;
 
+    /**
+     * @var Config
+     */
+    private $catalogConfig;
+
     /**
      * @param ResourceConnection $resourceConnection
      * @param LinkedProductSelectBuilderInterface $linkedProductSelectBuilder
      * @param CollectionFactory $collectionFactory
+     * @param Config $catalogConfig
      */
     public function __construct(
         ResourceConnection $resourceConnection,
         LinkedProductSelectBuilderInterface $linkedProductSelectBuilder,
-        CollectionFactory $collectionFactory
+        CollectionFactory $collectionFactory,
+        Config $catalogConfig
```

Comment:
This change is not backward compatible. Please make this parameter optional with fallback of getting this object instance from objectManager